pwndbg/gdblib/prompt.py

```diff
@@ -89,15 +89,4 @@ def set_prompt() -> None:
         prompt = "\x01" + prompt + "\x02"  # SOH + prompt + STX
 
     gdb.execute(f"set prompt {prompt}")
-
-
-if pwndbg.gdblib.events.before_prompt_event.is_real_event:
     gdb.prompt_hook = initial_hook
```

Comment:
Shouldn't this line be unindented here? If not removed (probably not)?
```suggestion
gdb.prompt_hook = initial_hook
```

because otherwise it will be in the `set_prompt` function defined above.